packages/editor/src/utils/media-upload/index.js

```diff
@@ -41,18 +54,39 @@ export default function mediaUpload( {
 		typeof currentPost?.id === 'number'
 			? currentPost.id
 			: currentPost?.wp_id;
+	const setUploadLock = () => {
```

Comment:
Nit: I'd call this `setSaveLock` or something, this name sounds like it's locking uploading :)

Reply:
thanks for the naming tip! updated in [`b9ce6f1` (#41120)](https://github.com/WordPress/gutenberg/pull/41120/commits/b9ce6f1312b23b5381e65c2685154c981b895958)